Add `bem decl intersect` command (close #219)

scf2k opened the pull request.

Commit:
- bem decl intersect added (close #219)

arikon commented:
Предлагаю научиться использовать `hub pull-request -i <issue>` для создания pull request про одно issue.

arikon commented on lib/techs/deps.js.js:
```diff
@@ -250,6 +250,19 @@ var Deps = exports.Deps = INHERIT({
     },
 
+    intersect: function(deps) {
+        var items1 = this.items, items2 = deps.items, newItems = {};
```
Code style:

``` js
var items1 = this.items,
    items2 = deps.items,
    newItems = {};
```

arikon commented:
Не хватает тестов.

Всю новую функционально надо стараться покрыть тестами, невзирая на то, что соседние команды тестами не покрыты (это наш промах).

Commit:
- tests for decl subtract/intersect added (reference #219)
- deps.js code formatted

arikon committed:
Add `bem decl intersect` command (ref #219)

- Test 3 declarations in .intersect() and .subtract() tests
- Fix tests code style
